lms/templates/courseware/proctored_exams_status.underscore

```diff
@@ -0,0 +1,4 @@
+<div class="exam-timer">
+You're taking <a href="<%= exam_url_path %>"> <%= exam_display_name %> </a> exam as a proctored exam. Good Luck!
```

Comment:
Actually, can you ask around at Arbisoft about how to I18N'ify this? I'm not sure myself.